src/platforms/stlink/platform.h

```diff
@@ -180,11 +192,13 @@ extern bool debug_bmp;
 #define SWO_UART_RX_PIN GPIO10
 
 /* This DMA channel is set by the USART in use */
-#define SWO_DMA_BUS    DMA1
-#define SWO_DMA_CLK    RCC_DMA1
-#define SWO_DMA_CHAN   DMA_CHANNEL5
-#define SWO_DMA_IRQ    NVIC_DMA1_CHANNEL5_IRQ
-#define SWO_DMA_ISR(x) dma1_channel5_isr(x)
+#define SWO_DMA_BUS     DMA1
+#define SWO_DMA_CLK     RCC_DMA1
+#define SWO_DMA_CHAN    DMA_CHANNEL5
+#define SWO_DMA_IRQ     NVIC_DMA1_CHANNEL5_IRQ
+#define SWO_DMA_ISR(x)  dma1_channel5_isr(x)
```

Comment:
These want running through clang-format to fix the formatting.

Reply:
Notice first commit doing exactly that. Because with contents of second commit of this patchset, pre-commit clang-format hook requires modifying entire block. Should it have 1 or 2 spaces between longest macro and values?

Reply:
Should be a single space.

Reply:
Dropped commit doing that.

Reply:
Huhh, so.. that made the lint pass angry - apparently, unlike anywhere else in the code where it uses one space, it wants two here. If you could please add it back in, we'll resolve this and the comment below as it actually really wants two spaces.

We can look at seeing why clang-format is doing that later.

Reply:
Restored.